app/Models/Contest.php

```diff
@@ -54,4 +54,14 @@ public function getCSVExport()
 
         return buildCSV($data);
     }
+
+    /**
+     * Set the sender attribute for the contest.
+     *
+     * @param string $value
+     */
+    public function setSenderAttribute($value)
+    {
+        $this->attributes['sender'] = ! empty($value) ? $value : null;
```

Comment:
No worries! Yep. That happened in a prior PR along w/ the migrations for the Messages :dancer: